docs/_data/menu-sql.yaml

```diff
@@ -242,6 +242,8 @@
     - text: Functions
       url: sql-ref-functions.html
       subitems:
+      - text: Built-in Functions
+        url: sql-ref-functions-builtin.html
```

Comment:
Okay .. now I understood why you included the file and why it's tricky. We should exclude `sql-ref-functions-builtin.html` conditionally but it's difficult because the generated file is md file.

We can't easily directly generate `sql-ref-functions-builtin.html` either because Jekyll adds some links and htmls when it converts md to html.

Reply:
Maybe we should keep `sql-ref-functions-builtin` as:

```
{% for static_file in site.static_files %}
    {% if static_file.name == 'agg_funcs.html' %}
### Aggregate Functions
        {% include_relative agg_funcs.html %}
#### Examples
        {% include_relative agg_funcs_examples.html %}
        {% break %}
    {% endif %}
{% endfor %}

{% for static_file in site.static_files %}
    {% if static_file.name == 'window_funcs.html' %}
### Windows Functions
        {% include_relative window_funcs.html %}
#### Examples
        {% include_relative window_funcs_examples.html %}
        {% break %}
    {% endif %}
{% endfor %}

...
```

Reply:
`mkdocs` should be able to produce a pretty HTML from MD for examples as well. Let me take another look tomorrow night or a day after tomorrow .. and push some more changes ...

Reply:
Ah, I see and I didn't have that idea. ok, I'll wait for your more updates. Thanks for the work, anyway.